src/content/protocols/compound-v3.md

```diff
@@ -0,0 +1,154 @@
+| Configurator Implementation | setTargetReserves                     | Sets the target reserves amount. Once the protocol reaches this amount of reserves of base asset, liquidators cannot buy collateral from the protocol.                                                                          | 0x0                            |
+| Configurator Implementation | addAsset                              | Adds an asset to the protocol through governance.                                                                                                                                                                               | 0x0                            |
+| Configurator Implementation | updateAsset                           | Modifies an existing asset’s configuration parameters.                                                                                                                                                                          | 0x0                            |
+| Configurator Implementation | updateAssetPriceFeed                  | Updates the price feed contract address for a specific asset.                                                                                                                                                                   | 0x0                            |
+| Configurator Implementation | updateAssetBorrowCollateralFactor     | Updates the borrow collateral factor for an asset in the protocol.                                                                                                                                                              | 0x0                            |
+| Configurator Implementation | updateAssetLiquidateCollateralFactor  | Updates the liquidation collateral factor for an asset in the protocol.                                                                                                                                                         | 0x0                            |
+| Configurator Implementation | updateAssetLiquidationFactor          | Updates the liquidation factor for an asset in the protocol.                                                                                                                                                                    | 0x0                            |
+| Configurator Implementation | updateAssetSupplyCap                  | Sets the maximum amount of an asset that can be supplied to the protocol. Supply transactions will revert if the total supply would be greater than this number as a result.                                                    | 0x0                            |
+| Configurator Implementation | transferGovernor                      | Changes the address of the Configurator’s Governor.                                                                                                                                                                             | 0x0                            |
```

Comment:
an implementation contract is stateless thus the scanner or etherscan show 0x0 as value, but please take the same as the proxy, thus timelock